pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -305,7 +305,6 @@ def progress_bar_metrics(self) -> Dict[str, float]:
         return self._progress_bar_metrics
 
     def teardown(self):
-        # TODO(@awaelchli): This should be handled by the loops themselves
```

Comment:
No longer applies? Why?

Reply:
You mentioned todos scattered around and this one is already mentioned in our follow up issue tracker #7938 , so I removed it